### PR TITLE
Maximum sentence complexity

### PR DESCRIPTION
**Pull request by LeonardPuettmannKern**
refinery
- [x] Tested by creator on refinery
- [x] Tested by reviewer on refinery
- [x] Ensured that output of brick conforms with refinery structure (to be checked by reviewer)

API
- [x] Tested by creator on localhost:8000/docs
- [x] Tested by reviewer on localhost:8000/docs

common code
- [x] Common code tested in notebook/ script by creator
- [x] Common code tested in notebook/ script by reviewer
- [x] Common code contains docstrings and type hints

additional points:
- [x] Docstring and README is existing
- [x] Import statements (in `__init__.py`)
- [ ] (If necessary) Added dependency to requirements.txt
- [ ] (If necessary) Added dependency to issue for refinery env [here](https://github.com/code-kern-ai/refinery/issues/166)
- [x] Published brick to Strapi CMS (locally)

Testing procedure: 
When testing in refinery, please ensure that the output of the brick conforms with the structure of refinery. 
For `extraction` bricks, this would be a tuple like `("label", span_start, span_end)`.
For `classification` bricks, this would be a string representing a label.
For `generator` bricks, this would either be a float, interger, string, boolean or a list, depending on the situation.

When testing the bricks, try to avoid using only one source of data. Meaning that you should not only use the `clickbait` sample
project, but also different texts with longer or more complex strings. 

A small refinery example project with a variation of texts called `bricks-test-data-project.zip` can be found in the bricks repository.

Implements #378

### CODE REVIEW

File: classifiers/reference_complexity/maximum_sentence_complexity/config.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        issue_id=378,
+        tabler_icon="ThreeDCubeSphere",
+        min_refinery_version="1.7.0",
+        state=State.PUBLIC.value,
+        type="python_function",
+        available_for=["refinery", "common"],
+        part_of_group=[
+            "reference_complexity",
+            "text_analysis"
+        ],  # first entry should be parent directory
```
Comment (FelixKirschKern) [inline classifiers/reference_complexity/maximum_sentence_complexity/config.py:19]: Also `question_complexity`

File: classifiers/reference_complexity/maximum_sentence_complexity/code_snippet_refinery.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    if score < 70:
+        return "standard"
+    if score < 80:
+        return "fairly easy"
+    if score < 90:
+        return "easy"        
+    return "very easy"
+
+if TARGET_LANGUAGE is not None:
+    textstat.set_lang(TARGET_LANGUAGE)
```
Comment (FelixKirschKern) [inline classifiers/reference_complexity/maximum_sentence_complexity/code_snippet_refinery.md:27]: suggest to move this up under the definition of `TARGET_LANGUAGE`. makes it easier to read the code

File: classifiers/reference_complexity/maximum_sentence_complexity/config.py (hunk 1 above)
Comment (LeonardPuettmannKern) [reply classifiers/reference_complexity/maximum_sentence_complexity/config.py]: Yes, I have added it. Although, as there is no actual group question_complexity containing bricks, it won't actually show the grouping in the frontend. It we just need this information from the backend, this is fine though.

File: classifiers/reference_complexity/maximum_sentence_complexity/code_snippet_refinery.md (hunk 2 above)
Comment (LeonardPuettmannKern) [reply classifiers/reference_complexity/maximum_sentence_complexity/code_snippet_refinery.md]: Sure, I've moved it!

**Review by FelixKirschKern — APPROVED** [review]

**Pull request merged**